code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -307,6 +307,12 @@ GLOBAL_VAR_INIT(cryo_overlay_cover_off, mutable_appearance('icons/obj/cryogenics
 
 			air1.temperature = clamp(air1.temperature - heat * delta_time / air_heat_capacity, TCMB, MAX_TEMPERATURE)
 			mob_occupant.adjust_bodytemperature(heat * delta_time / heat_capacity, TCMB)
+
+			//lets have the core temp match the body temp in humans
+			if(ishuman(mob_occupant))
+				var/mob/living/carbon/human/humi = mob_occupant
+				humi.adjust_coretemperature(humi.bodytemperature - humi.coretemperature)
```

Comment:
god this is the best/worst snake case ever (not your fault, just wanted to point it out)

Reply:
Ignore the fact that I named the proc ,`adjust_coretemperature`

Reply:
![anger](https://user-images.githubusercontent.com/42839747/100524466-b415c080-31b8-11eb-99e0-d97e410ac23f.jpg)